### PR TITLE
Update for Terraform 0.13

### PR DESCRIPTION
**Pull request by shoekstra**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  required_version = ">= 0.13"
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    datadog = {
+      source = "terraform-providers/datadog"
+    }
+  }
+}
```
Comment (svanharmelen) [inline versions.tf:11]: Is it needed to pit this in here? Shouldn’t this be in you own TF files that use this module instead?

Comment (shoekstra) [reply versions.tf:11]: That's what I thought, but without I get an error saying needs to be updated. I think it's needed for any providers not under the `hashicorp` or `-` namespace.

Comment (shoekstra) [reply versions.tf:11]:
```shell
- Finding latest version of hashicorp/datadog...

Error: Failed to install providers

Could not find required providers, but found possible alternatives:

  hashicorp/datadog -> terraform-providers/datadog

If these suggestions look correct, upgrade your configuration with the
following command:

The following remote modules must also be upgraded for Terraform 0.13
compatibility:
- module.datadog at
github.com/schubergphilis/terraform-aws-mcaf-datadog?ref=v0.1.0
```

Comment (svanharmelen) [reply versions.tf:11]: From the docs: `Each module must declare its own set of provider requirements, so if you have a configuration which calls other modules then you'll need to run this upgrade command for each module separately.` So seems like this is indeed needed 👍

Comment (svanharmelen) [reply versions.tf:11]: But I assume only the datadog one, right?  Not that it matters much to add them both...

Comment (shoekstra) [reply versions.tf:11]: We could I guess, I just ran the upgrade command and this is what I got. I think in most cases people will do the same and not manually specify provider sources, so there is some consistency there.

Comment (svanharmelen) [reply versions.tf:11]: Maybe we should run the update command on each module that we tough to make sure this file always follows that output then.

**Review by svanharmelen — APPROVED** [review]

File: versions.tf (hunk 1 above)
Comment (shoekstra) [reply versions.tf:11]: Definitely, two things to consider:

* Running it with an existing `versions.tf` will append the `required_providers` block and bump the `required_version` value, whereas running it with without an existing `versions.tf` have the `required_providers` block first and then `required_version`
* Any module we do this to will then require a minimum version of Terraform 0.13

Re potential inconsistency: easiest would be to indeed remove `version*.tf` from and run the upgrade command. Could potentially add this to CI in the future.

Re minimum version: I'm fine with this, I think a gentle push to upgrade and stay current is not a bad thing...

**Pull request merged**